joplin/base/models/official_documents_page.py

```diff
@@ -55,10 +57,14 @@ class OfficialDocumentPageOfficialDocument(Orderable):
 
     panels = [
         FieldPanel('date'),
-        FieldPanel('title'),
-        FieldPanel('authoring_office'),
-        FieldPanel('summary'),
-        FieldPanel('name'),
+        FieldPanel('title', widget=countMe),
+        FieldPanel('authoring_office', widget=countMe),
+        FieldPanel('summary', widget=widgets.CountableWidget(attrs={
```

Comment:
At first I wanted a cool and succinct way to override just the max_length of the default config I have set, but I don't think it's really worth that extra hassle.